cmat/trait_mapping/ols.py

```diff
@@ -33,13 +33,13 @@ def get_ontology_label_from_ols(ontology_uri: str) -> str:
     json_response = json_request(url)
 
     if not json_response:
-        return None
+        return ''
```

Comment:
Out of curiosity and nothing else, but why the change between return types now?

Reply:
This is actually an excellent question, and one that I'm now reconsidering...  The purpose was to avoid having [this join](https://github.com/EBIvariation/eva-opentargets/pull/382/files#diff-d45348d6909c9424f91184dc08636e4f2fbd089044342469a271db3ef711fb33R29) trip up on `None` values, and in my mind it didn't make much difference either way - but given your and @sundarvenkata-EBI's comment below I'm thinking it will be clearer to keep this returning `None` and just explicitly replace `None` with `''` for the string join.  This keeps the semantics clearer, i.e. `None` means we truly did not get a valid response from OLS, as opposed to if OLS for some reason returns an empty string (or even just whitespaces).